packages/web-components/src/components/rux-input/rux-input.tsx

```diff
@@ -170,13 +177,17 @@ export class RuxInput implements FormFieldInterface {
     @Watch('type')
     handleTypeChange() {
         this._setTogglePassword()
+        if (this.type === 'file') {
```

Comment:
Just in case they are changing to a type=file on the fly, this will handle the attribute mapping.